dist/accordion/css/accordion.min.css

```
@@ -1 +1,62 @@
-.aria-accordion__panel[hidden]{display:none}
\ No newline at end of file
+/* For legacy browsers support */
+.aria-accordion__panel[hidden] {
+  display: none;
+}
+
+/* For accordion panels with a transition effect */
+.aria-accordion__panel--transition {
+  display: block;
+  max-height: 100vh;
+  visibility: visible;
```

```suggestion
  visibility: visible;
  opacity: 1;
  height: auto;
```

Added `opacity` CSS property for a gradual appearance/disappearance of the content panel.
Added `height` CSS property to ensure a default height when the panel opened.